Sources/BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -83,6 +83,11 @@ public class CryptoStore: ObservableObject {
       }
     }
   }
+  public var origin: URLOrigin? {
```

Comment:
can we add a comment this is a dapp's origin?

Reply:
Added in [084d05c](https://github.com/brave/brave-ios/pull/7470/commits/084d05c615b0b0bf277b3bb1132d0202a73febcc)